dist/gcc-compatible/Hacl_P256.h

```diff
@@ -218,16 +272,37 @@ Hacl_P256_ecdsa_verif_without_hash(
   
  The last extract is taken from : https://neilmadden.blog/2017/05/17/so-how-do-you-validate-nist-ecdh-public-keys/
 */
-bool Hacl_P256_verify_q(uint8_t *pubKey);
+bool Hacl_P256_validate_public_key(uint8_t *pubKey);
 
 /*
- There and further we introduce notions of compressed point and not compressed point. 
-  
- We denote || as byte concatenation. 
-  
- A compressed point is a point representaion as follows: (0x2 + y % 2) || x.
+Validate a private key, e.g. prior to signing.
+
+Input: scalar: uint8[32].
   
- A not Compressed point is a point representation as follows: 0x4 || x || y.
+ Output: bool, where true stands for the scalar to be more than 0 and less than order.
+*/
+bool Hacl_P256_validate_private_key(uint8_t *x);
+
+
+/*****************************************/
+/* Point representations and conversions */
+/*****************************************/
+
+/*
+  Elliptic curve points have 2 32-byte coordinates (x, y) and can be represented in 3 ways:
+
+  - "raw" form (64 bytes): the concatenation of the 2 coordinates, also known as "internal"
+  - "compressed" form (33 bytes): the first byte is either  or , followed by x
```

Comment:
Missing the byte value here, gets removed from the comment during extraction? Same in the C file.

Reply:
nice catch Victor, thanks